src/services/sources.service.js

```diff
@@ -0,0 +1,28 @@
+var request = require('superagent');
```

Comment:
Can this file use es6 syntax?

Reply:
Updated

Reply:
The file is still using  module.exports and require.